examples/plot_simulate_evoked.py

```diff
@@ -36,10 +38,51 @@
 
 ###############################################################################
 # Let us first create our network from the params file and visualize the cells
-# inside it.
+# inside it. The default behaviour of Network is to add and instantiate six
+# 'default' drives, but we will suppress that by setting the
+# `initialise_hnn_drives`-argument to `False`.
 net = Network(params)
 net.plot_cells()
 
+###############################################################################
+# The network of cells is now defined, to which we add external drives as
+# required. Weights are prescribed separately for AMPA and NMDA receptors
```

Comment:
Perhaps add here a bulleted list with the possible drives that can be added (the ones you add in whats_new.rst) with links to the API page?

Reply:
Good idea, not sure how to implement the links in Sphinx though. Any pointers?

Reply:
done